packages/opencensus-core/test/test-root-span.ts

```diff
@@ -169,6 +184,7 @@ describe('RootSpan', () => {
       for (const span of root.spans) {
         assert.ok(span.ended);
       }
+      assert.equal(root.numberOfChildren, 1);
```

Comment:
there is no need to check for child count in other test case unless the test case affects the logic.

Reply:
Sure, Done.